shapely/tests/legacy/test_create_inconsistent_dimensionality.py

```diff
@@ -187,7 +187,7 @@
 def test_create_from_geojson(geojson):
     # exact error depends on numpy version
     with pytest.raises((ValueError, TypeError)) as exc:
-        wkt = shape(geojson).wkt
+        wkt = shape(geojson).wkt  # NOQA
```

Comment:
Why is this NOQA needed in this case?

Reply:
because of the unused `wkt` assignment (we can also remove that variable, which I guess was put there for debugging purposes)